### DISCUSSION TITLE
Void sprites - repairing ships?

### DISCUSSION THREAD

**Discussion by movingpictures** [endless-sky/endless-sky#8475] [Balance, Mechanics, and Interactions]
I do not know if this was intended, but testing out 0.10.0, at the stage of Remnant storyline where a gasship goes to Nenia and there are Korath there...

the little-wee puffin was ambushed by a Palavret (which my fleet devoured), but a void sprite came along and repaired the Puffin.

I don't know whether that was intended or not. Personally, I found it surprising and quite cool but the rational part of me wonders if that's attributing too much intelligence to the individual void sprites.

**Comment by Zitchas** [comment]
Maybe they just gave your ship the proverbial kick to get it restarted on the way by...

Definitely not intended, but cool enough I'm kind of inclined to leave it. If only as a semi-secret way to help player's that get stuck there.
